term_proposal/evan11401/project.md

```diff
@@ -0,0 +1,84 @@
+# High Dynamic Range Imaging using Multiple Exposure Images with Tone Mapping
+![img1](./img1.jpg)
+## Motivation
+The real world is high dynamic range, but the camera is limited dynamic range. In case to get the image that looks more like the real world image, high dynamic range imaging (HDRI) is invented.
+
+HDRI is widely used in the imaging and photography industry, such as the APPLE iPhone's camera also use this to get a better image.
+
+## Target
+I have read some tutorial to teach people how to implement the HDRI algorithm, but most of the function is provided by the OpenCV library.
+
+I know the library is powerful, but I think if I can do it on my own, I will learn the details and understand what it works. Moreover, I want to design the architecture like moving the time-consuming computation into c++, creating a user interface that users can easily use this program, and then merge it together.
```

Comment:
In contrast to the anecdotal saying, our job as engineers does include reinventing the wheels.  I sense that, no matter how OpenCV realized the algorithm, you would like to design the architecture by doing it from bottom up.

I think it will be a good attempt.  You may probably find a good way to make a more applicable implementation.  But it's also possible that you show that OpenCV has done a great job architecting it.  Either way, you will learn a lot about the algorithm and the software design.